packages/flutter_version_manager/lib/src/ui/widgets/optional_update_dialog.dart

```diff
@@ -0,0 +1,62 @@
+import "package:flutter/material.dart";
+import "package:flutter_version_manager/utils/scope.dart";
+import "package:version_repository_interface/version_repository_interface.dart";
+
+///
+Future<bool> defaultOptionalUpdate(
+  BuildContext context, {
+  required VersionCompatibiliy compatibility,
+  required bool backendLeading,
+}) async {
+  if (backendLeading) {
+    var config = VersionManagerScope.of(context).config;
```

Comment:
extract these 2 paths as functions to increase readability.

Reply:
shall I do the same structure as I did in mandatory_update_dialog.dart?